### PR TITLE
Helm chart: better document the purpose of the logLevel option

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

File: helm/nessie/README.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ $ helm uninstall --namespace nessie-ns nessie
 | livenessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed. Minimum value is 1. |
 | livenessProbe.terminationGracePeriodSeconds | int | `30` | Optional duration in seconds the pod needs to terminate gracefully upon probe failure. Minimum value is 1. |
 | livenessProbe.timeoutSeconds | int | `10` | Number of seconds after which the probe times out. Minimum value is 1. |
-| logLevel | string | `"INFO"` | The default logging level for the nessie server. |
+| logLevel | string | `"INFO"` | The minimum log level for the Nessie server. If you need to debug Nessie, set this to DEBUG, then add the following to the advancedConfig section: quarkus.log.category."<logger>".level: DEBUG where <logger> is the logger you want to debug. E.g. to debug issues with the Nessie configuration, add: quarkus.log.category."io.smallrye.config".level: DEBUG |
```
Comment (snazy) [inline helm/nessie/README.md:207]:
```suggestion
| logLevel | string | `"INFO"` | The minimum log level for the Nessie server. If you need to debug Nessie, set this to DEBUG, then add the following to the advancedConfig section: `quarkus.log.category."<logger>".level: DEBUG` where `<logger>` is the logger you want to debug. E.g. to debug issues with the Nessie configuration, add: `quarkus.log.category."io.smallrye.config".level: DEBUG` |
```

File: helm/nessie/values.yaml [hunk 2]
```diff
@@ -12,7 +12,12 @@ image:
   # -- The path to the directory where the application.properties file should be mounted.
   configDir: /deployments/config
 
-# -- The default logging level for the nessie server.
+# -- The minimum log level for the Nessie server. If you need to debug Nessie, set this to DEBUG,
+# then add the following to the advancedConfig section:
+# quarkus.log.category."<logger>".level: DEBUG
```
Comment (snazy) [inline helm/nessie/values.yaml:17]:
```suggestion
#   quarkus.log.category."<logger>".level: DEBUG
```

File: helm/nessie/values.yaml [hunk 3]
```diff
@@ -12,7 +12,12 @@ image:
...
+# where <logger> is the logger you want to debug. E.g. to debug issues with the Nessie
+# configuration, add:
+# quarkus.log.category."io.smallrye.config".level: DEBUG
```
Comment (snazy) [inline helm/nessie/values.yaml:20]:
```suggestion
#   quarkus.log.category."io.smallrye.config".level: DEBUG
```

File: helm/nessie/README.md (hunk 1 above)
Comment (adutra) [reply helm/nessie/README.md:207]: This file is generated.

**Review by snazy — APPROVED** [review]

**Pull request merged**